### DISCUSSION TITLE
Welcome, Laura Bigoni!

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#235] [Subscribe]
Ahoy @LauraBigoni, welcome aboard!

Thank you for joining us! Every day there are new quiz questions posted in [Discussions](https://github.com/ParanoidUser/yolo/discussions/categories/galaxy-trivia), you can try to answer them to earn the Galaxy Brain badge. Spread the word, collaborate and have fun, have fun, have fun! 😊

Let me know if you have any questions.

**Comment by LauraBigoni** [comment]
Thank you 🙏🏻
